core/src/test/kotlin/org/stellar/anchor/auth/AuthHelperTest.kt

```diff
@@ -22,8 +22,8 @@ class AuthHelperTest {
   @ParameterizedTest
   @EnumSource(AuthType::class)
   fun `test AuthHeader creation based on the AuthType`(authType: AuthType) {
-    val calendarSingleton = mockk<Calendar>(relaxed = true)
     lockAndMockStatic(Calendar::class) {
```

Comment:
Should this be replaced by the annotation?

Reply:
Probably not. I think ```val calendarSingleton = mockk<Calendar>(relaxed = true)``` should happen before the mock.

Reply:
Didn't notice that 👍